src/cmd/initContainer.go

```diff
@@ -115,6 +116,12 @@ func init() {
 		"Create a user inside the toolbox container whose login name is USER")
 	initContainerCmd.MarkFlagRequired("user")
 
+	flags.IntVar(&initContainerFlags.gid,
```

Comment:
The flags are sorted in one way and the flag definition other way.

Reply:
Good catch, should be fixed now.